app/browser/tabs.js

```diff
@@ -1015,6 +1016,7 @@ const api = {
           createProperties.isolated_storage = true
           createProperties.parent_partition = ''
           createProperties.tor_proxy = 'socks5://127.0.0.1:9050'
+          createProperties.tor_path = getExtensionsPath('tor') + '/tor'
```

Comment:
it is better to use `getExtensionsPath('bin')` so that we can have different binary in it like `geth`
and also need to check platform for Windows in order to use `tor.exe`